sdk/lib/atl/atlwin.h

```diff
@@ -587,20 +587,31 @@ class CWindow
     }
 
 #ifdef __ATLSTR_H__
-    UINT GetDlgItemText(int nID, CSimpleString& string)
+    UINT GetDlgItemText(int nID, CSimpleString& strText) const
     {
         HWND item = GetDlgItem(nID);
+        if (!item)
+        {
+            strText.Empty();
+            return 0;
+        }
+#if 1
+        return CWindow(item).GetWindowText(strText);
```

Comment:
This would allow avoiding the copy-paste implementation (see below) adapted from the `GetWindowText(CSimpleString& strText)` one I added further down.

Reply:
The class is simple enough, go for it.